taxonomies/ingredients.txt

```diff
@@ -13698,6 +13698,11 @@ sl:modificiran koruzni škrob
 sr:modifikovani kukuruzni skrob
 sv:modifierad majsstärkelse
 
+<en:modified starch
```

Comment:
I think we could add "modified" and "partially modified" in ingredients_processing.txt, that way we could (eventually) remove all the related entries in ingredients.txt.

But let's merge this PR first, we can do it later.